index.js

```diff
@@ -3,30 +3,69 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+  const result = []
+  for (let i = 0; i <= rolls.length - 1; i++) {
+    rolls.push(i);
+    return true;
```

Comment:
Due to where this return lies, this line is going to be hit and stop coding, return `true` every time.